msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -695,5 +695,27 @@ Define the properties your app uses to post a user activity feed.
 ```
 
 ***
->
->
+
+## defaultInstallScope
+
+**Required** - string
+
+It helps you to specify the default install scopes for the apps that you publish.
+
+| **Name** | **Type** | **Maximum size** | **Required** | **Description** |
+|----------|----------|------------------|--------------|-----------------|
+| `enum` | array of strings | 4 | ✔ | Specifies whether the scope offers an experience in the context of an individual user alone (personal), a team, a groupchat, or meetings.|
+| `description` | string | 128 characters | ✔ | The description that appears to users to indicate the purpose of this command. | 
```

Comment:
Description is not available as property to set by developer, it's actually part of manifest schema definition.